app/src/main/res/layout/administrator_controls_download_permissions_view.xml

```diff
@@ -73,7 +73,7 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginStart="40dp"
-        android:layout_marginEnd="4dp"
+        android:layout_marginEnd="@dimen/administrator_controls_download_permissions_view_upper_switch_compat_margin_end"
```

Comment:
I suggest `administrator_controls_download_permissions_view_widget_switch_compat_margin_end` instead
Ditto for others.